about.html

```diff
@@ -0,0 +1,157 @@
+            <span class="bar"></span>
+            <span class="bar"></span>
+          </div>
+          <div class="logo">
+            <a href="index.html"
+              ><img src="./assets/images/logo.PNG" alt="merctech mentorship"
+            /></a>
+          </div>
+          <nav class="nav-menu">
+            <ul class="menu-links">
+              <li>
+                <a href="index.html" class="a-links">Home</a>
+              </li>
+              <li>
+                <a href="#program" class="a-links">Program</a>
+              </li>
+              <li>
+                <a href="#join" class="a-links">Join</a>
+              </li>
+              <li>
+                <a href="#sponsor" class="a-links">Sponsor</a>
+              </li>
+              <li>
+                <a href="#news" class="a-links">News</a>
+              </li>
+              <li class="campaign">
+                <a href="#campaign" class="a-links">campaign</a>
+              </li>
+            </ul>
+          </nav>
+        </div>
+      </div>
+    </header>
+    <div class="header-contents-about">
+      <div class="container-herooo">
+        <div class="landing-text-about">
+          <div>
+            <img
+              src="./assets/images/theme1.PNG"
+              class="hero-img"
+              alt="theme"
+            />
+          </div>
+          <div class="about-text">
+            <p class="text-about">
+              Potential is equally distributed; opportunity is not. MENTOR aims to drive equity 
+              and close the mentoring gap through quality mentoring relationships for young people.
+              We activate a diverse cross-sector movement that prioritizes 
+              relationships 
+            </p>
+          </div>
+          <div class="contact-about">
+            <p>
+              Please contact us per Email for nay further questions about Bridge gap mentorship.
+            </p>
+            <p class="email">bridgegap@mentoring.org</p>
+          </div>
+        </div>
+      </div>
+    </div>
+    <section class="logo-info">
+        <p class="creative-logo">Bridge the Gap mentorship logo</p>
+        <p class="title-underline"></p>
+
+        <p class="logocc">
+          The logo of Bridge the Gap mentorship was decided through the logo
+          competition from 8, nov to 7, dec.
+        </p>
+        <div class="aboutlg">
+          <img src="./assets/images/about_logo 1.png" />
+        </div>
+    </section>
+    <section class="previous-summit">
+        <div class="previous-summ-theme">
+          <p>see past mentorship programs</p>
+        </div>
+        <div class="summ-theme-text">
+            <p>Take a look at the past two mentorships events that took place in Nairobi</p>
+        </div>
+        <div class="previous-img">
+            <img src="./assets/images/about_2011 1.png" alt="image1">
+            <img src="./assets/images/about_2011 1.png" alt="image2">
+        </div>
```

Comment:
This is good 👍🏽 but you should try with an overlay so that it can get closer to this 👇🏽 

![Screenshot 2023-03-18 at 14 42 43](https://user-images.githubusercontent.com/45739963/226129742-7b98377c-1cd5-413d-af57-5c35c42aaca6.png)

This is what you have 👇🏽 look to also adjust the title `See thee past ...` to make it fit on one line and fit the design.

![Screenshot 2023-03-18 at 14 43 18](https://user-images.githubusercontent.com/45739963/226129775-46d948cd-c784-44c4-b15a-5189fc54e19b.png)